docs/editors-guide/quality-control-tests.md

```diff
@@ -1570,18 +1628,20 @@ ORDER BY ?entity
 ###  qc-xref-syntax.sparql
 
 ```
-# home: hp
-prefix hasDbXref: <http://www.geneontology.org/formats/oboInOwl#hasDbXref>
-prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
-prefix owl: <http://www.w3.org/2002/07/owl#>
-prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+# # Invalid Xref
+#
+# **Problem:** A database_cross_reference is not in CURIE format.
+#
+# **Solution:** Replace the reference with a [CURIE](https://www.w3.org/TR/2010/NOTE-curie-20101216/).
 
-SELECT DISTINCT ?entity ?property ?value WHERE 
-{
-  ?entity hasDbXref: ?x .
+PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 
-  FILTER( regex(STR(?x), " ") || regex(STR(?x), ";") || STR(?x) = ""  )
-  BIND(hasDbXref: AS ?property)
+SELECT DISTINCT ?entity ?property ?value WHERE {
+  VALUES ?property {oboInOwl:hasDbXref}
+  ?entity ?property ?value .
+  FILTER (!regex(?value, "^[A-Za-z_][A-Za-z0-9_.-]*[A-Za-z0-9_]:[^\\s]+$"))
+  FILTER (!isBlank(?entity))
```

Comment:
@matentzn is this query only identifying triples where the value is not in a CURIE format? I'm confused by [Line 1635](https://github.com/monarch-initiative/mondo/pull/7759/files#diff-1491216dbf421ae0f0f312436d72bdc63a5331ebfdc1ed4819a7c44e9c059946R1635) that says `**Solution:** Replace the reference with a [CURIE]...`

Reply:
yes, 

```
 FILTER (!regex(?value, "^[A-Za-z_][A-Za-z0-9_.-]*[A-Za-z0-9_]:[^\\s]+$"))
```

Is a regex that checks that the xref is in CURIE format!

So the test is looking for all cross references where the value does not correspond to curie format.

```
FILTER (!isBlank(?entity))
```

reduces the query to annotations on classes, rather than axiom annotations like xrefs on synonyms.

Reply:
Gotcha, the query only identifies these cases (which is what I thought). The `**Solution:**` line makes it seem like the query was intended to fix the cases where a CURIE is not present.